partner_identification/tests/test_res_partner.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+from odoo.exceptions import ValidationError
+from .fake_models import ResPartner, setup_test_model, teardown_test_model
+
+
+class TestResPartner(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        setup_test_model(cls.env, ResPartner)
+        bad_cat = cls.env['res.partner.id_category'].create({
+            'code': 'another_code',
+            'name': 'another_name',
+        })
+        cls.env['res.partner.id_number'].create({
```

Comment:
@tarteo `cls.env['res.partner.id_number']` and `cls.env['res.partner.id_category']` use multipal time, IMO you can create global variable.

Reply:
I rather keep the code not related to migration as is.